Updated App Icons

QuiteYellow opened the pull request:
Updated app icons to support dark mode, light mode and tinted.

Commit:
Updated app icons

Updated app icons, now supports light, dark and tinted variants.

Commit:
Update .gitignore

jackweekes commented:
Regarding:

ITMS-90032: Invalid Image Path - - No image found at the path referenced under key 'CFBundleAlternateIcons': 'AppIcon2'

Is this something in the build settings on your end?

Walfrosch92 commented:
Thanks, there was a mistake in my plist :-) It should be reviewed by Apple soon and available on TestFlight.

Commit:
Added coloured labels and sort A-Z

I've added the colour labels from Mealie, amended the label text colour based on contrast, and sorted the Labels A-Z.

I've also changes the bottom padding.

jackweekes commented:
Can this one be accepted, please? I'll then put in a pull request that adds label colours 🙂

<img width="250px" src="https://github.com/user-attachments/assets/56e21dbd-78ad-48b0-b639-b970cddcd53b">

Commit:
Fixed ordering, applied label colour to chips

Fixed ordering, applied label colour to chips.

Commit:
Added complete shopping toggle

Added complete shopping toggle.

jackweekes commented:
It would be great if you could officially merge the commits you agree too, that way there's a proper history of what has and hasn't been accepted. Thanks.
